lib/components/Tabs/style.scss

```diff
@@ -43,9 +44,10 @@ $lineBottomHeight: 2px;
 	&__indication {
 		display: inline-block;
 		vertical-align: middle;
-		border-radius: 50%;
-		width: 18px;
-		height: 18px;
+		border-radius: 0.5 * $size;
```

Comment:
I gather that `20px` sound eval to `10px`, but just thinking of making this a `calc(#{$size} / 2)` so that if a future person updates it to say `21px` the half pixel would be handled by the browser.